core-web/apps/dotcms-ui/src/app/portlets/dot-categories/dot-categories-list/dot-categories-list.component.ts

```diff
@@ -12,8 +15,41 @@ import { DotCategoriesListStore, DotCategoriesListState } from './store/dot-cate
 export class DotCategoriesListComponent {
     vm$: Observable<DotCategoriesListState> = this.store.vm$;
     selectedCategories: DotCategory[] = [];
+
+    @ViewChild('dataTable')
+    dataTable: Table;
+
     constructor(private store: DotCategoriesListStore) {}
 
+    /**
+     * Add category in breadcrumb
+     * @param {DotCategory} category
+     * @memberof DotCategoriesListComponent
+     */
+    addBreadCrumb(category: DotCategory) {
+        this.dataTable.filter(category.inode, 'inode', null);
+        this.dataTable.filter(null, 'global', null);
+        this.store.addCategoriesBreadCrumb({ label: category.categoryName, id: category.inode });
```

Comment:
This is not a good approach, you are faking it trigger a navigation inside the table, but you have the state in the store and you can modify that.

Also, you are updating the state more than one time unnecessarily.

Please be careful with triggering extra re-renderings when you can do the whole thing in one operation.